build/NuSpecs/build-nupkg.ps1

```diff
@@ -137,6 +138,15 @@ if(-not $SkipFrameworkPackage)
 
     $NugetArgs = "$CommonNugetArgs -OutputDirectory $OutputDir\FrameworkPackage"
 
+    if(-not $SkipMakeNugetPackageAppxPackages)
```

Comment:
It looks like we now have two ways of creating the framework packages: MakeAllAppx.cmd and MUX-MakeFrameworkPackages-Steps.yml. They are both doing pretty much the same thing, but we should rationalize things. E.g. the yml should call a common script instead of having an embedded script.

Reply:
Is this something that I/we should do in this PR? I have not much experience working with azure jobs, so I am not sure how much I can help (mainly because I am not sure what the paths in the VM will be to run the scripts).

Reply:
If you don't feel comfortable making this change, leave things as they are for now, but file a separate issue to track rationalizing this in the future. It is not an urgent issue, so we can just add it to the backlog.

Reply:
I will try to change the MUX-MakeFrameworkPackages-Steps.yml file tomorrow, so that we don't pollute the backlog unnecessarily. After all we are already touching this area with this PR.

Reply:
Sounds good. Thanks Marcel!

Reply:
I've updated the step to now use MakeAllApxx.cmd with the correct parameters, I hope that the path to the script is right but I think it should work.

Reply:
Thanks for making the change!